marketplace_builder/views/pages/tutorials/partner-portal/updating-instance-configuration.liquid

```diff
@@ -92,12 +92,24 @@ You can enter multiple emails, comma-separated.
 "test_email": "sales@platform-os.com,developer@platform-os.com,support@platform-os.com"
 ```
 
+##### timeout_in_minutes
+
+This settings allows for changing user session duration. If not set, default timeout is 1 year. If you want log out users after 15 minutes of inactivity set it as in the example below:
+
+**Default value:** `1 year`
```

Comment:
![image](https://user-images.githubusercontent.com/546845/53696420-1c0e5b80-3dc7-11e9-905d-9a738712b897.png)

Reply:
I updated the default value. Different converters gave me a different result for 1 year in minutes, so let's double-check if this is the actual value we use. Thank you!